sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/text/TextFileFormat.scala

```diff
@@ -40,6 +40,8 @@ class TextFileFormat extends TextBasedFileFormat with DataSourceRegister {
 
   override def shortName(): String = "text"
 
+  override def toString: String = shortName.toUpperCase
```

Comment:
I did see it but I'd do the opposite and rather fix JSON and CSV than repeat what's in `shortName`. I even thought about defining the `toString` in a supertype, but could find nothing that would be acceptable.